packages/react-components/react-toolbar/src/components/Toolbar/useToolbar.ts

```diff
@@ -64,9 +64,23 @@ export const useToolbar_unstable = (props: ToolbarProps, ref: React.Ref<HTMLElem
     },
   );
 
+  const handleRadio: ToggableHandler = useEventCallback(
+    (e: React.MouseEvent | React.KeyboardEvent, name?: string, value?: string, checked?: boolean) => {
```

Comment:
```suggestion
    (e: React.MouseEvent | React.KeyboardEvent, name: string, value: string, checked?: boolean) => {
```
name and value should not be optional here or in props

Reply:
In the props it's already not optional, I guess that this is also true for `handleToggleButton` so I removed the optional there also. https://github.com/microsoft/fluentui/pull/25343/commits/49066edae68c01207a0140973e93892b468b2c5e